std/socket.d

```diff
@@ -1921,45 +1921,47 @@ static if (is(sockaddr_un))
     class UnixAddress: Address
     {
     protected:
-        sockaddr_un* sun;
-        socklen_t len;
-
+        struct
+        {
+        align (1):
+            sockaddr_un sun;
+            char unused; // placeholder for a terminating '\0'
+        }
 
         this() pure nothrow @nogc
         {
+            sun.sun_family = AF_UNIX;
+            sun.sun_path[].uninitializedFill('?');
```

Comment:
This doesn't make sense. `sun_path` is a `byte[]`, `uninitializedFill` is only needed for structs with copy ctors, and for cases where the data is uninitialized (in this case, it will be initialized).

I don't think we need a specialized function here, just:

```D
sun.sun_path[] = '?';
```

And remove the import for `uninitializedFill`